core/shared/src/main/java/de/mm20/launcher2/search/location/Departure.kt

```diff
@@ -0,0 +1,26 @@
+package de.mm20.launcher2.search.location
+
+import android.graphics.Color
+import de.mm20.launcher2.serialization.ColorSerializer
+import de.mm20.launcher2.serialization.DurationSerializer
+import de.mm20.launcher2.serialization.LocalTimeSerializer
+import kotlinx.serialization.Serializable
+import java.time.Duration
+import java.time.LocalTime
+
+@Serializable
+data class Departure(
+    @Serializable(with = LocalTimeSerializer::class)
+    val time: LocalTime,
```

Comment:
I think this should be a `LocalDateTime`? Because it refers to one singular trip (including delay), not a recurring trip. Also we should clarify: is this the scheduled time or the real time including delays?